### DISCUSSION TITLE
Use more than one editor

### DISCUSSION THREAD

**Discussion by jmkingsf** [vueup/vue-quill#92] [Q&A]
Does Vue-Quill support more than one editor on a page? I'm having trouble finding any documentation on Vue-Quill specifically.

Thanks.

**Comment by fylzero** [comment]
@jmkingsf I was able to implement this with multiple editors on a single page.  Quill is one of the few editors that seems to handle this well.

**Comment by jmkingsf** [comment]
I was able to get this working as well. Turns out both the toolbar and the editor just need unique ids and then it will work.
